source/module_io/berryphase.cpp

```diff
@@ -395,10 +395,10 @@ void berryphase::Berry_Phase(int nbands,
                              const K_Vectors& kv)
 {		
 	std::complex<double> cave = 0.0;
-	double *phik = new double[total_string];
+    std::vector<double> phik(total_string);
```

Comment:
As I said, I consider this kind of changes as "one place" to replace 'new' with vector, not "three places"